src/graphql-types/ReasonApolloQuery.re

```diff
@@ -74,12 +93,14 @@ module Get = (Config: ReasonApolloTypes.Config) => {
       apolloData##fetchMore({
         "variables": variables,
         "query": graphqlQueryAST,
+        "updateQuery": apolloData##updateQuery,
       }),
     networkStatus: apolloData##networkStatus,
   };
   let make =
       (
         ~variables: option(Js.Json.t)=?,
+        ~updateQuery: option((Config.t, updateQueryOptions) => Config.t)=?,
```

Comment:
This is tricky, because `Config.t` is the initial Query, but you should be able to pass any query in the updateQuery?

Reply:
I'm not sure if that is correct. My understanding is that `updateQuery` isn't the best name, it's not a new query it's just a function to combine the results from the initial query and a `fetchMore` query. The fetchMore query is the original query but can have new variables. 

I'm getting this information from: https://www.apollographql.com/docs/react/features/pagination.html I may be interpreting it incorrectly.

Reply:
In the link there is a mention: 
```
note this is a different query than the one used in the Query component
```

Reply:
updateQuery shouldn't be a prop of the Query component. It's only a property of the config o ject passed to fetchMore